### PR TITLE
Connect leaderboard to Safira

### PR DESCRIPTION
**Pull request by ruioliveira02**
I am sure what to do if the request fails. Should the user be redirected to another page, should a message pop up?

### CODE REVIEW

**Comment by nelsonmestevao** [comment]
> I am sure what to do if the request fails. Should the user be redirected to another page, should a message pop up?

I think in most cases, the best way to handle it is to show an error pop up. When the error in safira is a good message, just show it to the user. If the message is a bad error, just 'Something went wrong', sorry not sorry :D 

You could use just https://headlessui.dev/react/dialog. I guess is the best way using headlessui. You could also use https://www.npmjs.com/package/react-notifications, but it is not as good-looking in my opinion.

**Comment by ruioliveira02** [comment]
I implemented an error pop up. It is not much, so if you want it changed, say so

**Comment by nelsonmestevao** [comment]
image


Login page and dashboard have those lines, and we cannot interact with the page :/

File: pages/attendee/leaderboard.js [hunk 1]
```diff
@@ -60,75 +65,55 @@ function addDate(date, days) {
...
+  const [hallOfFame, updateHallOfFame] = useState(false);
+  const [date, updateDate] = useState(defaultDate);
+  const [leaderboard, updateLeaderboard] = useState([]);
+  const [error, updateError] = useState(false);
 
-  const testUser = {
-    name: "John Doe",
-    badges: 42,
-  };
+  useEffect(() => requestLeaderboard(), [leaderboard]);
```
Comment (miguelbrandao) [inline pages/attendee/leaderboard.js:94]: The `[leaderboard]` here is (probably) inducing a bug of multiple requests

**Review by miguelbrandao — CHANGES_REQUESTED** [review]
There is something wrong with the styling
image

**Comment by miguelbrandao** [comment]
> image
> 
> Login page and dashboard have those lines, and we cannot interact with the page :/

@nelsonmestevao I cant reproduce in either Chrome or Firefox

**Review by miguelbrandao — APPROVED** [review]

**Pull request merged**